Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -344,10 +322,15 @@ static int __dl_iterate_phdr(DlIteratePhdrCallbackFunction callback, void* data)
 
 static void initialize_libc(DynamicObject& libc)
 {
+    uintptr_t stack_guard = get_random<uintptr_t>();
```

Comment:
We should be able to use the ELF AT_RANDOM auxv here instead of calling out to the get random syscall.